core/domain/subtopic_page_domain_test.py

```diff
@@ -23,6 +23,74 @@
 import utils
 
 
+class SubtopicPageChangeDomainUnitTests(test_utils.GenericTestBase):
+    """Tests for subtopic page change domain objects."""
+    user_id = 'user_id'
+    topic_id = 'topic_id'
+    subtopic_id = 1
+
+    def setUp(self):
+        super(SubtopicPageChangeDomainUnitTests, self).setUp()
+        self.subtopic_page_change = subtopic_page_domain.SubtopicPageChange({
+            'cmd': subtopic_page_domain.CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY,
+            'subtopic_id': 1,
+            'property_name': 'page_contents_html',
+            'new_value': 'a',
+            'old_value': 'b'
+            })
+
+    def test_init_method(self):
+        with self.assertRaisesRegexp(Exception, 'Invalid change_dict *'):
+            subtopic_page_domain.SubtopicPageChange({
+                'subtopic_id': 1,
+                'property_name': 'page_contents_html',
+                'new_value': 'a',
+                'old_value': 'b'
+            })
+        with self.assertRaisesRegexp(Exception, 'Invalid change_dict *'):
```

Comment:
Divide these assertions into multiple tests i.e, each method/function should test one functionality. You can go through [the wiki page for more guidelines](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests).

Reply:
Hi @DubeySandeep I named the functions for init method